spec/fixtures/eurofxref-daily.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`vcr` is just helper but not necessary. Anyway we should probably add comment with source url where is this file from.